src/library-authoring/LibraryLayout.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import LibraryAuthoringPage from './LibraryAuthoringPage';
+import { LibraryProvider } from './common/context';
+
+const LibraryLayout = () => (
```

Comment:
:+1: This will help also with the SearchContext

Reply:
Thanks!